UTOL-En Using the Attendance Function in UTOL (for Course Instructors and TAs) (#1468)
utelecon/utelecon.github.io: src/pages/en/utol/lecturers/attendances/index.mdx

2. During the class, present the one-time password to enrolled students. Some specific methods are listed below.

    - In face-to-face classes, writing the one-time password in large letters in an empty space on the blackboard allows enrolled students to submit attendance without rushing.

    - In online classes, prepare a slide with the password in large letters. Additionally, writing it in the corner of other slides allows students to check it again if they fail to submit it successfully.

    - If you share the one-time password via Zoom chat, students can easily copy and paste it. However, be aware that if they need to re-enter Zoom due to connection issues, they will not be able to check the previous chat history.

3. Instruct enrolled students to submit the one-time password. Additionally, confirm that they have submitted their attendance.

    - Please note that the procedure for submitting attendance from the students’ side is as follows.

    <details>
    <summary>Procedure for submitting attendance from the enrolled students’ side</summary>

>     1. If enrolled students open or reload the Course Top screen after the “Available date and time for submission”, the “Submit Attendance” button will be displayed.

Reviewer: UIに合わせて
```suggestion
    1. If enrolled students open or reload the Course Top screen after the “Available date and time for submission”, the “Submit attendance” button will be displayed.
```

        - This button is not displayed on the instructor’s screen.

        - If students open the Course Top screen before the “Available date and time for submission,” the button will not be displayed even when the time comes. In this case, please reload the Course Top screen.

        ![](./img/otp.drawio.png)

>     2. When enrolled students click the “Submit Attendance” button, they are asked to enter a one-time password. If they enter the correct password and submit it, their attendance will be recorded on UTOL.

Reviewer: ```suggestion
    2. When enrolled students click the “Submit attendance” button, they are asked to enter a one-time password. If they enter the correct password and submit it, their attendance will be recorded on UTOL.
```

        ![](./img/otpin.png)

    </details>

#### Troubleshooting
{:#trouble-otp}

<details>

> <summary>“Submit Attendance” button is not displayed</summary>
> 
> - The “Submit Attendance” button is not displayed on the instructors’ or TAs’ “editing” screens, unlike on enrolled students’ screens.

Reviewer: ```suggestion
<summary>“Submit attendance” button is not displayed</summary>

- The “Submit attendance” button is not displayed on the instructors’ or TAs’ “editing” screens, unlike on enrolled students’ screens.
```

- If you are using automatically registered attendance management, please configure the setting by following: “[Edit the attendance management automatically registered through the class session function > Display automatically registered attendance management on the enrolled students’ screens](#set-auto-attendance)”.

</details>

<details>
<summary>I don’t know the one-time password</summary>

- You can check the one-time password by opening the attendance confirmation screen through the following steps: “[Manually configure the attendance management > Manually add or modify the attendance management for each session](#set-each-attendance-b)”.

</details>

### Course instructors configure attendance management in batches

> By using the “Batch attendance data registration” function, instructors can register attendance in batches.

Reviewer: UIと単語の対応を見るに、ここは「出席者一括登録」が対応するので、これがより正確な訳の気がします。
```suggestion
By using the “Batch attendee of registration” function, instructors can register attendance in batches.
```

1. Prepare a participants’ list in a text file. Available formats are the following three methods: “Specify User ID”, “Specify Student ID”, and “Card reader format”.

    - “Specify User ID”: Write one Common ID (10-digit number) per line.

    - “Specify Student ID”: Write one Student ID number per line.

    - “Card reader format”: Write in “10-digit common id,2000/01/01” format per line. Files output by the loaned card reader follow this format.

    ![](./img/attcheck_UL_li.drawio.png)

2. On the Course Top screen, click on the ![≡](../../_icons/hamburger.png){:.icon} icon located in the upper left corner to display the menu.

3. From the menu, expand the “Course Content >” and click on “Manage Attendance”.

    ![](./img/attman.drawio.png)

4. The “Manage Attendance” screen will be displayed. Click the ![︙](../../_icons/control_btn_icon.png){:.icon} button next to the line of the course you want to check and click “Register Attendance.”

    ![](./img/attmanmenu3.drawio.png)

> 5. Upload the file from the “Batch attendance of registration” field.

Reviewer: UIを見るにおそらくこうかと思います
```suggestion
5. Upload the file from the “Batch attendee of registration” field.
```

    ![](./img/attmanmenu3.drawio.png)

4. Click the “Download format” displayed in blue in the “Batch attendance data upload” section in the middle of the “Manage Attendance” screen.

    ![](./img/attcheck_DL_ed.png)

5. “Password Setting” dialog will appear. Set a password and click the “Download” button.

    ![](./img/attcheck_DL_pw.png)

6. A password-protected file that can be opened in Excel will be downloaded. Edit downloaded file.

    - Column E (Attendance Status: Present) to Column I (Attendance Status: Excused Absent) correspond to the attendance status of each enrolled student. To set an attendance status, enter half-width “1” in the appropriate cell, and leave all other cells blank.

7. Click the “Reference” button in the “Batch attendance data upload” section in the middle of the “Manage Attendance” screen, select the edited file, and then click the “Upload” button.

    ![](./img/attcheck_UL_ed.drawio.png)

8. Enter the password and click the “Upload” button.

9. After uploading, the updated attendance status will be displayed. Click the “Confirm” button.

>     ![](./img/attcheck_DL_pw.png)

Reviewer: おそらくスクリーンショットの指定が間違っています
```suggestion
    ![](./img/attcheck_UL_pw.png)
```
